src/utilities/spacing/demo/directions.twig

```diff
@@ -0,0 +1,22 @@
+{% if step < 0 %}
+  {% set step = 'n' ~ step|abs %}
+{% endif %}
+
+{% embed '../../../objects/rhythm/rhythm.twig' with {
+  class: 'demo-u-' ~ name ~ '-directions'
```

Comment:
Nit: When we were doing similar includes in Liquid, we somehow documented the variables that were passed into the include, in a form that looked a bit like JSDoc. It would just be nice to know what `name` is / can be, in this case, and that it's a required "param" for the include.

Reply:
This seems like overkill for templates created for demonstration purposes.

Reply:
Fair enough.